_episodes/02-practice-learning.md

```diff
@@ -112,10 +112,9 @@ If you choose to use an analogy, be sure its usefulness outweighs its potential
 > ## Analogy Brainstorm
 >
 > 1. Think of an analogy to explore. Perhaps you have a favorite that relates to your area of professional interest, or a hobby. If 
-> you prefer to work with an example, consider this common analogy from education: "teaching is like gardening."
+> you prefer to work with an example, consider this common analogy from education: "teaching is like gardening".
```

Comment:
based on the [style guide](https://docs.carpentries.org/topic_folders/communications/resources/style-guide.html#quotation-marks) I'm actually not acturally sure we should move this outside, since "teaching is like gardening." is a full sentence

Reply:
Yes, I see you point @brownsarahm - in which case we might need two fullstops to end the outer sentence? 
"'.. you prefer to work with an example, consider this common analogy from education: "teaching is like gardening.".'

Reply:
I don't think we put two either. the one full stop does double work in this case.